### PR TITLE
Fix certs

### PR DESCRIPTION
**Pull request by RichieAHB**
This PR will drop the requirement for having to set Chrome to ignore localhost https certificates. I'm also presuming it will also mean you can test thrashers in Firefox more easily.

The PR creates certificates for localhost and drops them into a `certs` directory in the root of the project and uses these for `grunt-contrib-connect`.

There's a requirement on Homebrew that's been added in this PR but hopefully that's a worthwhile price to pay to fix this in all browsers forever 🤞

Tagging @guardian/guardian-design-team

### CODE REVIEW

**Comment by mchv** [comment]
@RichieAHB love it great work 🏅

File: setup-certs.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+
+CERT_CIRECTORY="certs"
```
Comment (akash1810) [inline setup-certs.sh]: there's a small typo here
```suggestion
CERT_DIRECTORY="certs"
```

File: setup-certs.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+echo "Creating certificate with mkcert - hold on 🚀"
+
+brew install mkcert nss
+
+# setup rootCA, if this exists it will not overwrite
+mkcert -install
+
+mkdir -p $CERT_CIRECTORY
```
Comment (akash1810) [inline setup-certs.sh]:
```suggestion
mkdir -p $CERT_DIRECTORY
```

File: setup-certs.sh [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+mkcert -key-file=./$CERT_CIRECTORY/localhost-key.pem -cert-file=./$CERT_CIRECTORY/localhost.pem localhost
```
Comment (akash1810) [inline setup-certs.sh]:
```suggestion
mkcert -key-file=./$CERT_DIRECTORY/localhost-key.pem -cert-file=./$CERT_DIRECTORY/localhost.pem localhost
```

**Pull request merged**